plugin.video.wizjatv/main.py

```diff
@@ -6,25 +6,28 @@
 PY2 = sys.version_info[0] == 2
 if PY2:
     from urlparse import parse_qs
+    from urllib import urlencode
 else:
-    from urllib.parse import parse_qs
+    from urllib.parse import parse_qs, urlencode
 
 from ptw.debug import log_exception, log, start_trace, stop_trace, TRACE_ALL
 import wizja
 
+# Addon Internal Modes:
+PLAY_LINK = 6
```

Comment:
Nie lubię magic-numbers. proponuję ich unikać. Właściwie to może być dowolny tekst ale na razie zostawiłem super wartość `6` :-) Tyle, że w kodzie jest używana nazwa.